### ISSUE TITLE
Project description in repository is gender specific

### ISSUE THREAD

**Issue by jamieide** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#279]
The project description "FizzBuzz Enterprise Edition is a no-nonsense implementation of FizzBuzz made by serious **businessmen** for serious business purposes." uses gender specific language which violates the Contributor Code of Conduct. Change "businessmen" to "businesspeople" or similarly inclusive language.

**Comment by emiln** [comment]
Hello, @jamieide and thank you for shining a light on this issue.

The current situation is certainly unacceptable, and the responsible text author has been severely reprimanded. Suffice to say that their employment has been terminated and their name soiled in various blog posts that strongly imply, but do not directly disclose, their identity.

The question remains what to do about the exact phrasing of the project description. We have put our trusted Inclusivity Consultant on the case, and a few points have been raised:

*   The phrasing should be inclusive regardless of perceived membership of any race or species, which makes "people" problematic. Surely we do not want to oppress those who socially, physically, or otherwise affiliate as otherkin.
*   The section delineating the purpose of FizzBuzz Enterprise Edition ("for serious business purposes") suggest wrongful conduct in using FizzBuzz Enterprise Edition for other purposes than the one mentioned. This is problematic for an inclusive approach to the _utility_ of the project.
*   The repeated use of such phrases as "no-nonsense" and "serious" imposes a general sense of restriction regarding the ways in which users are allowed to _feel_ during their interactions with the project. This is highly problematic as we impose a rigid set of authoritative rules and requirements about the feelings, thoughts, and states of being on anyone interacting with the project.

In conclusion this issue is of high concern, but requires further deliberation to solve in a satisfying and inclusive manner. Thank you for bringing this up.
